### ISSUE TITLE
Test Set

### ISSUE THREAD

**Issue by bdytx5** [Woodman718/FixCaps#7]
Hello, I just wanted to confirm that the results presented (96pct acc) were on the ham10000 test set, as opposed to a custom split of the training set?

Thanks,
Brett

**Comment by Woodman718** [comment]
https://github.com/Woodman718/FixCaps/blob/main/augmentation/split.ipynb

**Comment by bdytx5** [comment]
Thanks for the response. 

I did see the script for splitting the data, however its still unclear to me exactly what data you use for your reported results. 

Specifically, the code references val525s8 and test525, but I am unsure as to where this data came from, as well as if you ever benchmarked the model on the official test set, as opposed to a subset of the training set. Without using the official test set, these results cannot be compared to other models evaluated on the official test set. 

Can you reference where you get this data? 

Thanks,
Brett

**Comment by Woodman718** [comment]
While you may not have had a chance to review my code, the data is sourced from 10015 (ISIC2018).
At the Top of the README:
https://aistudio.baidu.com/aistudio/datasetdetail/151696
This version exhibits poor stability, and I've been working on addressing related issues in Version 2.  
You can find the improvements and fixes on [GitHub](https://github.com/Woodman718/CapsNets).  
The repository also includes my responses to relevant questions.
In the Citation section at the bottom of the README(CapsNet), you'll find my most recent paper, dated June 23.

**Comment by bdytx5** [comment]
Ok, thanks for clarifying!
